packages/synapse-react-client/src/components/Markdown/MarkdownCollapse.tsx

```diff
@@ -71,8 +70,8 @@ export const MarkdownCollapse = (props: MarkdownCollapseProps) => {
       <Collapse in={show}>
         <div id="collapse-text">
           <MarkdownSynapse
-            onMarkdownProcessingDone={onMarkdownProcessingDone}
             {...props}
+            onMarkdownProcessingDone={onMarkdownProcessingDone}
```

Comment:
In Storybook, they provide mock implementations for all props.  Make sure we use the onMarkdownProcessingDone defined in this component